ui/src/shared/reducers/appConfig.js

```diff
@@ -0,0 +1,20 @@
+import {AUTOREFRESH_DEFAULT} from 'src/shared/constants'
+
+const initialState = {
+  autoRefresh: AUTOREFRESH_DEFAULT,
+}
+
+const appConfig = (state = initialState, action) => {
```

Comment:
@cryptoquick and I discussed this a bit. My opinion was that appUI is for UI-only state (presentation mode), whereas appConfig would be data persisted about user choices for configuring the app. UI and config feel separate to me. Thoughts?